src/Liquifier.sol

```diff
@@ -204,6 +205,15 @@ contract Liquifier is Initializable, UUPSUpgradeable, OwnableUpgradeable, Pausab
         return eEthShare;
     }
 
+    // Swap user's eETH for Liquifier's stETH
+    function swapEEthForStEth(uint256 _amount) external whenNotPaused nonReentrant {
```

Comment:
Remove this?
<img width="334" alt="Screenshot 2024-08-07 at 10 48 16 PM" src="https://github.com/user-attachments/assets/193b440e-9ba7-4a3b-bae2-77874d63f0a3">